### PR TITLE
Basic structural changes

### PR DESCRIPTION
**Pull request by upulieh**

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
 import Navbar from "./components/Navbar";
 import Content from "./components/Content";
+import "./App.css";
 
-const baseURL = "http://localhost:3001"; //move to .env
-
+// after authentication remove Main from /
 function App() {
-  const [loading, setLoading] = useState(false);
+  const [loading, setLoading] = useState(false); //use this for something?
```
Comment (vongohren) [inline frontend/src/App.js:16]: Hehe this is a nice way of making the user understand that the server is doing business and you have a state that it is currently loading.

But you are setting and managing what shall happen to it

File: frontend/src/pages/Orders.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+                    }
+                  </th>
+                ))
+              }
+            </tr>
+          ))
+        }
+      </thead>
+      {/* Apply the table body props */}
+      <tbody {...getTableBodyProps()}>
```
Comment (vongohren) [inline frontend/src/pages/Orders.js:94]: This is very very complex to read and potentially add alot of complexity to when you want to go off the paht for any reason

Comment (vongohren) [reply frontend/src/pages/Orders.js:94]: It might be more helpful to split up in components

Comment (upulieh) [reply frontend/src/pages/Orders.js:94]: Thanks I'll check it out. Yes it definitely did add to the complexity.

File: frontend/src/App.js (hunk 1 above)
Comment (upulieh) [reply frontend/src/App.js:16]: I'm thinking about adding some animation but didn't come across a proper one yet.

Comment (vongohren) [reply frontend/src/App.js:16]: @upulieh you can just use a spinner gif or a spinner css style. Google that :D 

Be mindfull of choices that provide value versus choices that are for neatness :D

Comment (upulieh) [reply frontend/src/App.js:16]: Thanks. I'll try

**Pull request merged**